### DISCUSSION TITLE
Add mute function

### DISCUSSION THREAD

**Discussion by grazikgafriginal** [kkrypt0nn/Python-Discord-Bot-Template#72] [Ideas]
Please add mute function

**Comment by kkrypt0nn** [comment]
Hello,

Thank you for this idea but I do not think this is needed at this moment in bots, considering it will need lots of engineering to make sure temporary mutes resume when the bot restarts - it would be overkill for a template.

There's the timeout function of Discord which makes the command a little bit useless in my opinion
![image](https://user-images.githubusercontent.com/43011723/193535095-b35d1bf8-c651-4f67-bf60-9706057ef6f8.png)

Might do it later but that's not my current priority, otherwise feel free to code it and make a PR for that

~ Krypton
